### ISSUE TITLE
The "view" link is visible when links have not been supplied

### ISSUE THREAD

**Issue by danielcgold** [ayastreb/jekyll-maps#33]
See screenshots!
![screen shot 2018-03-09 at 1 20 19 pm](https://user-images.githubusercontent.com/1226984/37222881-a8025af6-239c-11e8-9659-ab3dcf315f53.jpg)

![screen shot 2018-03-09 at 1 20 49 pm](https://user-images.githubusercontent.com/1226984/37222899-b7441784-239c-11e8-94fe-ca5681d50315.jpg)

**Comment by ayastreb** [comment]
Hi @danielcgold 
thanks for reporting this!
How do you provide locations - from page front matter or from data files?
Can you show how your `{% google_map %}` tag looks like?

**Comment by danielcgold** [comment]
No prob!

Yea, from front matter. It looks like this:

```

hong-kong.markdown

---
title: Hong Kong’s Hidden Photo Spots
date: 2017-07-10 16:15:00 Z
location:
- title: Choi Hung Rainbow Estate
  latitude: 22.334757
  longitude: 114.20734
- title: Ping Shek Estate
  latitude: 22.333634
  longitude: 114.209081
- title: Montane Mansion
  latitude: 22.28449
  longitude: 114.211968
- title: Lai Tak Tsuen
  latitude: 22.278229
  longitude: 114.194438
- title: Instagram Pier
  latitude: 22.289601
  longitude: 114.132743
---
```

and the google maps tag looks like this

```
{% google_map width="100%" %}
```

**Comment by ayastreb** [comment]
Sorry for the delay!
What is the use-case you're trying to achieve? Do you want to display a map with multiple markers, where each marker has a popup, but does not have a link?

Right now, if you specify locations in page front matter data - they all will have this page URL as popup URL. If you specify locations from data file and use it with `src` attribute - then popups will not have URLs.

**Comment by ayastreb** [comment]
I've just released v2.3.0, it does not use page url if there are multiple locations on the page now.

**Comment by danielcgold** [comment]
Woo!! 🎉
